templates/server-statefulset.yaml

```diff
@@ -106,6 +106,18 @@ spec:
         - name: consul
           image: "{{ default .Values.global.image .Values.server.image }}"
           env:
+            - name: ADVERTISE_IP
+              valueFrom:
+                fieldRef:
+                  {{- if not .Values.server.exposeGossipAndRPCPorts }}
+                  fieldPath: status.podIP
+                  {{- else }}
+                  # Server gossip and RPC ports will be exposed as a hostPort
```

Comment:
Should we use helm comments here so this doesn't render for users? Unless we think they should see it.
e.g.

```
  {{/* We set check_update_interval to 0s so that check output is immediately viewable
       in the UI. */}}

```

Reply:
Great comment!

Reply:
Thanks! And yup, switching it to helm comment.